pi_ldapproxy/bindcache.py

```diff
@@ -23,48 +23,54 @@ def __init__(self, timeout=5):
         :param timeout: Number of seconds after which the entry is removed from the bind cache
         """
         self.timeout = timeout
-        #: Map of tuples (dn, password) to insertion timestamps (determined using ``reactor.seconds``)
+        #: Map of tuples (dn, app_marker, password) to insertion timestamps (determined using ``reactor.seconds``)
         self._cache = {}
 
-    def add_to_cache(self, dn, password):
+    def add_to_cache(self, dn, app_marker, password):
```

Comment:
What is good pythonian?
Would we one day define a ``cacheObject`` instead of a tuple, which has the attributes
* dn
* app_markes
* password

and then define comparison methods?